docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/logo/logo.twig

```diff
@@ -16,6 +16,7 @@
  *    - breakpoint: [object] Object with 'mobile' and 'desktop' keys:
  *       - url: [string] URL for the logo.
  *       - alt: [string] Image alt text.
+ *       - svg_logo: [content] Svg Logo (Optional).
```

Comment:
@joshua-salsadigital 
Please rename the variable to `html` to be consistent with how SVGs are embedded in Social Links component. This property should contain SVG code ready for being embedded.